<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBalance">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listProducts">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboCoin">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsert">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Coin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuy">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy product</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdmin">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Product (Admin only)</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtLog">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Messages will appear here...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVending_Machine">
    <property name="title">
     <string>Vending Machine</string>
    </property>
   </widget>
   <addaction name="menuVending_Machine"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
